README.md

```diff
@@ -4,16 +4,24 @@ nxdk is a software development kit for the original Xbox. nxdk is a revitalizati
 It is maintained by the [XboxDev](https://github.com/XboxDev/XboxDev) community.
 
 Notable features:
+- Portable toolchain that works on modern versions of Windows, macOS and Linux.
 - No complicated cross-compiling or big library dependencies! Builds with `make` and just needs standard tools and llvm.
-- 3D graphics! nxdk includes and expands on pbkit, a library for interfacing with the Xbox GPU.
+- Modern C / C++ standards and compiler features.
+- Supports popular APIs like Windows API and BSD sockets.
+- SDL2 support for input, audio and 2D graphics.
+- Custom API for 3D graphics using NVIDIA-designed shader-languages (with additional Xbox extensions).
+- Open-Source drivers which can be modified to get the most out of the hardware.
+- Modifiable startup code, for as much system control as necessary.
+- Supported by an active community that can help with problems and responds to bug reports.
 
 Build Status
 ------
 ![CI Status for "Build Samples"](https://github.com/XboxDev/nxdk/workflows/Build%20Samples/badge.svg?branch=master&event=push)
 
 Status
 ------
-nxdk is currently in early stages of development.
+While nxdk still is in early stages of development, it can already be used in many projects.
```

Comment:
This sentence is weird. "It can be used in projects" or "It is already being used in many projects". Maybe also: "Useful for a variety of projects" or something.
But "can be used in many projects" sounds weird to me.